code/game/objects/structures/tables_racks.dm

```diff
@@ -590,11 +486,29 @@
 
 /obj/structure/table/reinforced/prison
 	desc = "A square metal surface resting on four legs. This one has side panels, making it useful as a desk, but impossible to flip."
-	icon_state = "prisontable"
+	icon = 'icons/obj/smooth_objects/prison_table.dmi'
+	icon_state = "prison_table-0"
+	base_icon_state = "prison_table"
 	table_prefix = "prison"
 
+/obj/structure/table/reinforced/fabric
+	name = "cloth table"
+	desc = "A fancy cloth-topped wooden table, bolted to the floor. Fit for formal occasions."
+	icon = 'icons/obj/smooth_objects/table_fabric.dmi'
+	icon_state = "table_fabric-0"
+	base_icon_state = "table_fabric"
+	table_prefix = "fabric"
+	parts = /obj/item/frame/table
+	reinforced = TRUE
+
+/obj/structure/table/reinforced/fabric/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	return FALSE
```

Comment:
?

Reply:
lame attempt to keep the table from being disassembled, removed now